lib/graphql/language/parser.rb

```diff
@@ -269,6 +275,7 @@ def definition
         else
           loc = pos
           desc = at?(:STRING) ? string_value : nil
+          comment = at?(:COMMENT) ? value : comment
```

Comment:
This code is not very easy to read, to be honest we added it because one of the tests was failing and this one worked.